src/content/upgrades/merge/issuance/index.md

```diff
@@ -15,79 +15,25 @@ The **issuance** of ETH is the process of creating ETH that did not previously e
 emoji=":chart_decreasing:"
 title="ETH issuance tldr">
 
-- Mining rewards ~13,000 ETH/day pre-merge
-- Staking rewards ~1,600 ETH/day pre-merge
-- **After The Merge, only the ~1,600 ETH per day will remain, dropping total new ETH issuance by ~90%**
-- The burn: At an average gas price of at least 16 gwei, at least 1,600 ETH is burned every day, which effectively brings net ETH inflation to zero or less post-merge.
+- Before The Merge ~13,000 ETH/day was issued to pay miners
+- After The Merge, issuance depends upon the number of active validators on the network, so varies slot to slot.
+- The total issuance is approximately 90% smaller now than it was pre-merge.
+- Ether is also burned in every slot, with a greater amount burned when gas is higher. This means ether can be inflationary or deflationary depending on network demand.
```

Comment:
Being a tldr section, my preference would be to keep some numbers for daily issuance under PoS available here, currently ~1700 ETH/day... I think it's helpful for people just looking for some simple numbers.

Reply:
idk I removed it implies the daily issuance is a fixed number - I find this to be misleading - its dependent upon N active validators and their performance. The original and your comment already vary by 100ETH/day.
I guess its ok with appropriate caveats, but I preferred the takeaway being a) issuance is variable, b) its about 1/10th what it was before merge.

Reply:
The ideal scenario here would be live numbers. Maybe a big numbers component + Dune API could be used in the future to solve this? Three numbers: an average PoW issuance at some point close to The Merge (maybe the month leading up to it?), previous days' PoS issuance, and percentage reduction.

As much as I dislike seeing things written this way, the trade-off here might be something like: "_The total issuance is approximately 90% smaller now than it was pre-merge (~1700ETH as of October 2022)._" 

What do you think @jmcook1186 @wackerow?

Reply:
fine by me